modules/wri_paragraph/config/install/field.field.paragraph.image_cta.field_block.yml

```diff
@@ -0,0 +1,27 @@
+langcode: id
+status: true
+dependencies:
+  config:
+    - block_content.type.image_cta
+    - field.storage.paragraph.field_block
+    - paragraphs.paragraphs_type.image_cta
+id: paragraph.image_cta.field_block
+field_name: field_block
+entity_type: paragraph
+bundle: image_cta
+label: Blok
+description: ''
+required: true
+translatable: true
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:block_content'
```

Comment:
Anything with `handler: 'default:block_content'` can be left untranslated -- the fields within the block reference can be translated instead.